_includes/live-stream-embed.html

```diff
@@ -0,0 +1,34 @@
+<div class="stream-container">
+  <video id="live" class="video-js vjs-default-skin vjs-big-play-centered" data-setup='{"fluid": true}'
+   controls preload autoplay loop>
+    <p class="vjs-no-js">To view this video please enable JavaScript, and consider upgrading to a web browser that supports HTML5 video</p>
+  </video>
+
+  <div class="stream-selector" id="StreamSelector">
+    <div id="SelectStream">
+      <h2>Select Live Stream Source</h2>
+      <div class="stream-selector-options">
+        <div id="ipfsStream">
+          <button class="stream-option ipfs-stream" title="Play the live stream on through a peer-to-peer hypermedia protocol. More at ipfs.io">
```

Comment:
I like the title, it is pretty subtle and missed it the first, is there another way to surface this?

Reply:
Like maybe a question mark/hover pattern?
or a "what's this?" that links to the repo with that sentence as an explainer or something,,,?

Reply:
Thinking through this and ended up with revising so that the text block shows below the option instead of the `title`. Does it make sense?
![screen shot 2018-07-10 at 10 21 12 am](https://user-images.githubusercontent.com/227587/42515975-05adf766-842b-11e8-96cd-848609cdaabe.png)

Reply:
I like it!